src/engine/controls/cuecontrol.cpp

```diff
@@ -883,6 +891,54 @@ void CueControl::hotcueSet(HotcueControl* pControl, double value, HotcueSetMode
         }
     }
 
+    // EveCue-Loop
+    bool TrackStem = false;
```

Comment:
bools should be named for building a "sentences" with if. Like `if (hasStems) {` or such. And start with a lower case.

Reply:
fixed